apollo-compiler/src/test/graphql/com/example/input_object_type/ReviewInputExpected.java

```diff
@@ -0,0 +1,59 @@
+package com.example.input_object_type;
+
+import java.lang.Integer;
+import java.lang.String;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+public final class ReviewInput {
```

Comment:
You might want to add more methods to it, like business logic related to your app.
I understand it might not be the best approach but I can see how people are gonna request this in the future for sure.
Fine to keep it this way for now, but it's gonna bite us in the future